### PR TITLE
lib: fix import

### PR DESCRIPTION
**Pull request by lucab**
This fixes the style of an import statement.

### CODE REVIEW

**Comment by lucab** [comment]
Note: this is a cosmetic PR which can be quickly landed, so that I can re-trigger CI on https://github.com/seanmonstar/httparse/pull/156 without having to wait for approval on each iteration.

**Review by seanmonstar — APPROVED** [review]

**Pull request merged**
